### PR TITLE
add description data nodes

### PR DESCRIPTION
**Pull request by boucadair**

### CODE REVIEW

File: yang/ietf-schedule.yang [hunk 1]
```diff
@@ -614,7 +629,6 @@ module ietf-schedule {
           "Corredponds to seven days of the week.";
```
Comment (QiufangMa) [inline yang/ietf-schedule.yang:629]:
```suggestion
          "Corresponds to seven days of the week.";
```

**Review by QiufangMa — COMMENTED** [review]
Adding the description node into generic-schedule-params might be good (as this is the common and generic parameter), but that way i don't feel it necessary to also be defined in the period and recurrence. Thoughts?

**Comment by boucadair** [comment]
> Adding the description node into generic-schedule-params might be good (as this is the common and generic parameter), but that way i don't feel it necessary to also be defined in the period and recurrence. Thoughts?

From an ops standpoint, description fields are very useful as this is a placeholder for recording contexts, etc. Parent "description" field might not be sufficient in all contexts.

**Comment by QiufangMa** [comment]
> > Adding the description node into generic-schedule-params might be good (as this is the common and generic parameter), but that way i don't feel it necessary to also be defined in the period and recurrence. Thoughts?
> 
> From an ops standpoint, description fields are very useful as this is a placeholder for recording contexts, etc. Parent "description" field might not be sufficient in all contexts.

Are you referring to description for each single recurrence/period entry? It seems okay to me then. I worry slightly that we are making the schedule yang definition huger and huger;-）

**Review by QiufangMa — APPROVED** [review]

**Pull request merged**